scss/utilities/_json-api.scss

```diff
@@ -18,8 +22,10 @@
 /// and the `export` mixin
 /// to generate JSON out of Sass variables.
 ///
-/// See the [`sass.jsonFile` configuration][json]
-/// to ensure that Herman has access
+/// Create a prefixed by `_` sass file, to avoid parsing it
+/// by mistake. You need compiled it to the CSS file set
+/// at [`sass.jsonFile` configuration][json].
+/// Make sure that Herman has access
```

Comment:
I don't think this is the right place to teach people about Sass partials, or when to use them. In either case, we _do_ want this file compiled, and the `sass.jsonFile` should point to the output CSS that is generated. I think an actual example would go farther towards communicating both this and the import, rather than trying to state them separately.

Reply:
Yes, boss!